py/images/demo.py

```diff
@@ -0,0 +1,169 @@
+        with sshd_config.open(encoding='utf8') as f:
+            config = f.read().splitlines(keepends=True)
+
+        changed = False
+        new_config = []
+        for line in config:
+            if re.match(r'^[#]?PasswordAuthentication no', line):
+                print("Enabling sshd password authentication")
+                line = 'PasswordAuthentication yes\n'
+                changed = True
+            new_config.append(line)
+
+        if changed:
+            with sshd_config.open('w', encoding='utf8') as f:
+                f.write(''.join(new_config))
+
+        else:
+            print("Not changing SSH config, it looks good")
+
+        target.chroot_run(['dpkg-reconfigure', 'openssh-server'])
+
+    def setup_liquid_sudo(self, target):
+        sudoers_liquid_demo = target.mount_point / 'etc/sudoers.d/liquid_demo'
+        with sudoers_liquid_demo.open('w', encoding='utf8') as f:
+            f.write('liquid ALL=(ALL:ALL) NOPASSWD: ALL\n')
+
+    def setup_network(self, target):
+        ens3_cfg = target.mount_point / 'etc/network/interfaces.d/ens3.cfg'
+        with ens3_cfg.open('w', encoding='utf8') as f:
+            f.write('auto ens3\niface ens3 inet dhcp\n')
+
+    def setup_console(self, target):
+        grub_files = [
+            'boot/grub/grub.cfg',
+            'etc/default/grub',
+            'etc/default/grub.d/50-cloudimg-settings.cfg',
+        ]
+        for rel_path in grub_files:
+            path = target.mount_point / rel_path
+            run(['sed', '-i ', 's/console=tty1/console=ttyS0/g', str(path)])
+
+    @contextmanager
+    def ansible_bind_mount(self, target):
+        local_setup_path = Path(__file__).resolve().parent.parent.parent
+        setup_path = target.mount_point / 'opt/setup'
+
+        assert not setup_path.exists()
+        setup_path.mkdir()
+        run(['mount', '--bind', str(local_setup_path), str(setup_path)])
+
+        try:
+            yield setup_path
+
+        finally:
+            run(['umount', str(setup_path)])
+            setup_path.rmdir()
+
+    def setup_ansible(self, target, domain):
+        if not Path('/usr/bin/ansible-playbook').is_file():
+            self.install_host_dependencies()
+
+        with self.ansible_bind_mount(target) as setup_path:
+            config_yml = setup_path / 'ansible/vars/config.yml'
+            with config_yml.open('w', encoding='utf8') as f:
+                f.write('liquid_domain: {}\n'.format(domain))
+                f.write('devel: true\n')
+
+            run([
+                'ansible-playbook',
+                'image_chroot.yml',
+            ], cwd=str(setup_path / 'ansible'))
+
+    def kill_testdata(self, target):
+        import_testdata = (
+            target.mount_point /
+            'opt/hoover/libexec/import_testdata'
+        )
+        with import_testdata.open('w') as f:
+            f.write('#!/bin/bash\necho "no testdata"\n')
+
+    def fix_hoover_ui(self, target):
```

Comment:
This shouldn't be needed if we first re-run ansible, right? Else, we need an issue for making sure we remove old build artifacts if the git repo changed.